device-types/DataDirect Networks/SS9012.yaml

```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: DataDirect Networks
+model: SS9012
+slug: ss9012
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`